src/Microsoft.Diagnostics.Tools.RuntimeClient/Eventing/EventPipeClient.cs

```diff
@@ -17,6 +17,33 @@
 
 namespace Microsoft.Diagnostics.Tools.RuntimeClient
 {
+    public enum EventPipeErrorCode : uint
```

Comment:
Feel free to check this in for now, but as part of our RuntimeClient API review I think we'll want to change these names. EventPipe is specific only to the CollectTracing/StopTracing APIs whereas these errors are shared for all commands. Using EventPipe in the name obfuscates that distinction.